packages/amplify-graphql-searchable-transformer/src/definitions.ts

```diff
@@ -40,8 +46,13 @@ const STRING_CONDITIONS = ID_CONDITIONS;
 const INT_CONDITIONS = ['ne', 'gt', 'lt', 'gte', 'lte', 'eq', 'range'];
 const FLOAT_CONDITIONS = ['ne', 'gt', 'lt', 'gte', 'lte', 'eq', 'range'];
 const BOOLEAN_CONDITIONS = ['eq', 'ne'];
-import assert from 'assert';
 
+export const AGGREGATE_TYPES = [
+  'SearchableAggregateResult',
+  'SearchableAggregateScalarResult',
+  'SearchableAggregateBucketResult',
+  'SearchableAggregateBucketResultItem',
```

Comment:
should we add input aggregate types too? I don't think it is necessary, just wanted to confirm.

Reply:
they are only necessary for the return types they cannot be applied for inputs